source/config/configSpec.py

```diff
@@ -179,6 +179,7 @@
 	allowSkimReadingInSayAll = boolean(default=False)
 	alertForSpellingErrors = boolean(default=True)
 	handleInjectedKeys= boolean(default=true)
+	maxRepeatedKeyPressDelay = integer(default=500, min=100, max=20000)
```

Comment:
A small thing. I am curious why this is named as it is?

iOS calls this feature "doubleTapTimeout". In our case, something like "multiPressTimeout" would seem more descriptive of what this actually is--not a delay, but a timeout.

"Key repetition delay", as written in the PR title, is never how I would have thought to search for a PR on this topic. In normal computing terms, key repetition delay is usually associated with how frequently a key is repeated when held down, which is not what this is. In fact, doing a web search for the phrase will take you right to its usual meaning.
I hate to be critical on such a minor point of important work, but I would regret not pointing it out in case you agreed. :)

```suggestion
	multiPressTimeout = integer(default=500, min=100, max=20000)
```